### PR TITLE
[DSS] [PEPPER-1180] Osteo Spanish translation of email html

### PR DESCRIPTION
**Pull request by aweng98**
[PEPPER-1180](https://broadworkbench.atlassian.net/browse/PEPPER-1180)

Add email templates for the Spanish language in study Osteo.

- Added new `*_es.html` email templates
- Updated ``study-events.conf` and `sendgrid_emails.conf` to include new `*_es.html` email templates

## Checklist

- [ ] I have labeled the type of changes involved using the `C-*` labels.
- [ ] I have assessed potential risks and labeled using the `R-*` labels.
- [ ] I have considered error handling and alerts, and added `L-*` labels as needed.
- [ ] I have considered security and privacy, and added `I-*` labels as needed
- [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [ ] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have added regression test labels to the jira ticket
- [ ] I have added verification steps to the jira ticket
- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

[PEPPER-1180]: https://broadworkbench.atlassian.net/browse/PEPPER-1180?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/osteo/study-events.conf [hunk 1]
```diff
@@ -706,8 +706,10 @@
       },
       "action": {
         "type": "SENDGRID_EMAIL",
-        "emailTemplate": ${emails.joinMailingList},
-        "language": "en",
+        "templates": [
+          { "emailTemplate": ${emails.joinMailingList}, "language": "en", "isDynamic": true },
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/study-events.conf]: Existing "en" email templates are not dynamic templates. They are old legacy templates.
"isDynamic" should be false for all existing "en" (english) templates.

Comment (aweng98) [reply pepper-apis/studybuilder-cli/studies/osteo/study-events.conf]: I see. I will fix this asap.

File: pepper-apis/studybuilder-cli/studies/osteo/sendgrid_emails.conf [hunk 2]
```diff
@@ -30,7 +38,7 @@
       "subject": "Reminder: Follow-up Surveys Available",
       "filepath": "emails/parental_about_child_reminder.html",
       "render": true,
-      "isDynamic": true
+      "isDynamic": false
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/sendgrid_emails.conf:41]: Please NOTE:
english should be false
spanish should be true

File: pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_child_contact_1wk_reminder_es.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    }
+  </style>
+</head>
+
+<body style="background: #ffffff; width: 100%; height: 100%; padding-top: 60px; padding-left: 60px;">
+<table style="width: 540px; padding: 0; -webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%; mso-table-lspace: 0pt; mso-table-rspace: 0pt; font-family: 'Source Sans Pro', sans-serif; border-collapse: collapse !important;"
+       border="0" cellpadding="0" cellspacing="0" width="100%">
+  <tr style="width: 540px;">
+    <td align="left" style="margin: 0; padding: 50px 0 0 0; font-family: 'Source Sans Pro', sans-serif;">
+      <a href="-ddp.baseWebUrl-" target="_blank" style="cursor: pointer; text-decoration: none;">
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_child_contact_1wk_reminder_es.html]: This should be dynamic convention {{baseWebUrl}}

File: pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_child_contact_1wk_reminder_es.html [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    </td>
+  </tr>
+  <tr style="height:0px">
+    <td style="border-left:solid #ffffff 1px;border-right:solid #ffffff 1px;border-bottom:solid #ffffff 1px;border-top:solid #ffffff 1px;vertical-align:top;background-color:#ffffff;padding:20px 0px 0px 0px">
+      <div>&nbsp; <a href="-ddp.baseWebUrl-/activity-link/{{activityInstanceGuid}}?participantGuid={{participant_guid}}" style="font-size:16px;font-family:Helvetica,Arial,sans-serif;font-weight:normal;color:#ffffff;text-decoration:none;background-color:#7154ff;border-top:15px solid #7154ff;border-bottom:15px solid #7154ff;border-left:25px solid #7154ff;border-right:25px solid #7154ff;border-radius:3px;display:inline-block"
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_child_contact_1wk_reminder_es.html]: This should be dynamic convention {{baseWebUrl}}

File: pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_email_verification_es.html [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Por ahora, haz clic a continuación para verificar tu dirección de correo electrónico. Te enviaremos un correo electrónico con más información sobre la renovación del consentimiento poco después de tu cumpleaños.</p>
+    </td>
+  </tr>
+  <tr style="height:0px">
+    <td style="border-left:solid #ffffff 1px;border-right:solid #ffffff 1px;border-bottom:solid #ffffff 1px;border-top:solid #ffffff 1px;vertical-align:top;background-color:#ffffff;padding:20px 0px 0px 0px">
+      <div>&nbsp; <a href="{{baseWebUrl}}/verify?invitation=-ddp.invitationId-&next=%2Fthank-you" style="font-size:16px;font-family:Helvetica,Arial,sans-serif;font-weight:normal;color:#ffffff;text-decoration:none;background-color:#7154ff;border-top:15px solid #7154ff;border-bottom:15px solid #7154ff;border-left:25px solid #7154ff;border-right:25px solid #7154ff;border-radius:3px;display:inline-block"
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_email_verification_es.html]: This should be dynamic convention {{baseWebUrl}}
Please search for -ddp in *_es.html files for such stuff missed

File: pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_next_steps_es.html [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+        <li style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Proporcionar actualizaciones sobre tus antecedentes médicos, lo que incluye centros o médicos nuevos que hayas visitado</li>
+      </ol>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Este proceso te llevará alrededor de 15 a 20 minutos.</p>
+    </td>
+  </tr>
+  <tr style="height:0px">
+    <td style="border-left:solid #ffffff 1px;border-right:solid #ffffff 1px;border-bottom:solid #ffffff 1px;border-top:solid #ffffff 1px;vertical-align:top;background-color:#ffffff;padding:20px 0px 0px 0px">
+      <div>&nbsp; <a href="{{baseWebUrl}}/accept?invitation=-ddp.invitationId-&next=%2Fdashboard" style="font-size:16px;font-family:Helvetica,Arial,sans-serif;font-weight:normal;color:#ffffff;text-decoration:none;background-color:#7154ff;border-top:15px solid #7154ff;border-bottom:15px solid #7154ff;border-left:25px solid #7154ff;border-right:25px solid #7154ff;border-radius:3px;display:inline-block"
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_next_steps_es.html]: This should be dynamic convention {{invitationId}}

File: pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_thank_you_es.html [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+        <img src="https://storage.googleapis.com/${assetsBucketName}/osteo/project-logo.png"
+             alt="Osteosarcoma Project logo" style="-ms-interpolation-mode: bicubic; border: 0; outline: none; text-decoration: none;"
+             width="230px" height="63px" border="0">
+      </a>
+    </td>
+  </tr>
+
+  <tr style="width: 540px;">
+    <td style="padding: 50px 0 0 0;">
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear -ddp.participant.firstName-,</p>
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_thank_you_es.html]: update to dynamic convention

File: pepper-apis/studybuilder-cli/studies/osteo/emails/self_consent_completed_es.html [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">i aún no ha proporcionado su dirección postal ni la información de contacto de sus médicos y hospitales, puede completar el proceso aquí:</p>
+    </td>
+  </tr>
+  <tr style="height:0px">
+    <td style="border-left:solid #ffffff 1px;border-right:solid #ffffff 1px;border-bottom:solid #ffffff 1px;border-top:solid #ffffff 1px;vertical-align:top;background-color:#ffffff;padding:20px 0px 0px 0px">
+      <div>&nbsp; <a href="{{baseWebUrl}}/activity-link/{{activityInstanceGuid}}?participantGuid=-ddp.participant.guid-" style="font-size:16px;font-family:Helvetica,Arial,sans-serif;font-weight:normal;color:#ffffff;text-decoration:none;background-color:#7154ff;border-top:15px solid #7154ff;border-bottom:15px solid #7154ff;border-left:25px solid #7154ff;border-right:25px solid #7154ff;border-radius:3px;display:inline-block"
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/emails/self_consent_completed_es.html]: This should be dynamic convention {{participantGuid}}
Please search for -ddp in *_es.html files for such stuff missed

File: pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_child_contact_1wk_reminder_es.html (hunk 3 above)
Comment (aweng98) [reply pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_child_contact_1wk_reminder_es.html]: good catch! Will search and fix all.

File: pepper-apis/studybuilder-cli/studies/osteo/emails/self_consent_completed_es.html (hunk 8 above)
Comment (ssettipalli) [reply pepper-apis/studybuilder-cli/studies/osteo/emails/self_consent_completed_es.html]: @aweng98 ^^ looks like this is missed
Please search for -ddp in ALL  *_es.html files for such stuff missed from your IDE

Comment (aweng98) [reply pepper-apis/studybuilder-cli/studies/osteo/emails/self_consent_completed_es.html]: hmm, can't believe i missed that. Just checking, if pattern was not changed for some variables like this, it is not going to break the app, right? Should we update them in all es *.html too?

Comment (aweng98) [reply pepper-apis/studybuilder-cli/studies/osteo/emails/self_consent_completed_es.html]: I searched for `-ddp.` in `emails` dir, I did not find any occurrence in any `*_es.html` files. It should be the only one I had missed.

**Review by ssettipalli — APPROVED** [review]
applied in dev and tested

**Pull request merged**